compiler/noirc_evaluator/src/ssa/checks/check_for_underconstrained_values.rs

```diff
@@ -32,6 +35,32 @@ impl Ssa {
             })
             .collect()
     }
+
+    /// Detect Brillig calls left unconstrained with manual asserts
+    /// and return a vector of bug reports if any have been found
+    pub(crate) fn check_for_missing_brillig_constraints(&mut self) -> Vec<SsaReport> {
+        // Skip the check if there are no Brillig functions involved
+        if !self.functions.values().any(|func| func.runtime().is_brillig()) {
+            return vec![];
+        };
+
+        let functions_id = self.functions.values().map(|f| f.id().to_usize()).collect::<Vec<_>>();
```

Comment:
Why are you converting function ID's to `usize` and then back to `FunctionId` in the loop?

Reply:
This is a leftover from copying some boilerplate of the previous check. Will fix it in both places.

Reply:
Fixed